be/src/runtime/load_channel_mgr.h

```diff
@@ -85,6 +88,7 @@ class LoadChannelMgr {
     std::mutex _lock;
     // load id -> load channel
     std::unordered_map<UniqueId, std::shared_ptr<LoadChannel>> _load_channels;
+    SpinLock _load_channels_lock;
     std::shared_ptr<LoadChannel> _reduce_memory_channel = nullptr;
```

Comment:
This lock's scope is same as _lock, I think we could just use _lock.

Reply:
done

Reply:
done